src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,26 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HomePage extends BasePage{
+    public HomePage(WebDriver givenDriver){
+        super(givenDriver);
+    }
+
+    //WebElements
+
+    By avatarIcon = By.cssSelector("img[class='avatar']");
+    By notificationMessage = By.cssSelector("div.success.show");
+    By firstPlaylist = By.cssSelector(".playlist:nth-child(3)");
+    By deleteBtn = By.cssSelector("button.del.btn-delete-playlist");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```